frontend/src/boot/socketio.js

```diff
@@ -3,7 +3,7 @@ import {Loading} from 'quasar'
 import { $t } from '@/boot/i18n'
 
 export default ({ Vue }) => {
-    var socket = io(`${window.location.protocol}//${window.location.hostname}:${process.env.API_PORT}`);
+    var socket = io(`${window.location.protocol}//${window.location.hostname}:${window.location.port != '' ? ':'+window.location.port : ''}`);
```

Comment:
Hi, I've tried your PR and it's a very useful feature. Right now you have a typo here - the colon the before port number is duplicated, if the port is non default, e.g. `https://localhost::8443`

Either the library or browser then strips the `::8443` and tries it on default port (`443` for https).

```suggestion
    var socket = io(`${window.location.protocol}//${window.location.hostname}${window.location.port != '' ? ':'+window.location.port : ''}`);
```
After this fix it looks good to me.  :shipit: